### DISCUSSION TITLE
What is the 'This device' device?

### DISCUSSION THREAD

**Discussion by f18m** [alandtse/alexa_media_player#2268] [Q&A]
Hi,
I'm using this wonderful integration. Everything works well with my 2 alexa devices.
However if I check with devices are created by the Alexa Media player integration I see 5 of them: 2 alexas, 1 for a samsung TV I own, and other 2 which are named:
* Everywhere
* This Device

I assume the "everywhere" is a pseudo device I can use to play a media on all other devices.
However, what is the "This Device?" ? 
The model is reported as 'VOX A2IVLV5VM2W81' but I don't recognize that... is it possible that 'This device' represents the media player associated with the audio output available on the miniPC where HomeAssistant runs?

**Comment by markuslaube** [comment]
"This device" is already listed by Alexa on Amazon pages as a kind of summary of Alexa apps. I stumbled across it myself as I was experimenting with different things.

In any case, the device is also your HA installation. But even if you have Alexa on your mobile phone. From my point of view, it is difficult to address as soon as you are not sure how many there are :-/

Source:
https://github.com/thorsten-gehrig/alexa-remote-control/commit/0dcc480f1ce9ed8f6a05096cb9a6606b205f9e6c

I hope this helps you

**Reply by f18m** [reply]
thanks @markuslaube , that helps!
